docs/build_info.md

```diff
@@ -0,0 +1,55 @@
+# account
+
+[<-Back-to:Auth](database-auth.md)
+
+**The \`build_info\` table**
+
+**Structure**
+
+| Field                 | Type         | Attributes | Key | Null | Default           | Extra          | Comment    |
+|-----------------------|--------------|------------|-----|------|-------------------|----------------|------------|
+| [build][1]            | INT          |            | PRI | NO   |                   |                | Identifier |
+| [majorVersion][2]     | INT          |            |     | YES  |                   |                |            |
+| [minorVersion][3]     | INT          |            |     | YES  |                   |                |            |
+| [bugfixVersion][4]    | INT          |            |     | YES  |                   |                |            |
+| [hotfixVersion][5]    | CHAR(3)      |            |     | YES  |                   |                |            |
+| [winAuthSeed][6]      | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [win64AuthSeed][7]    | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [mac64AuthSeed][8]    | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [winChecksumSeed][9]  | VARCHAR(40)  |            |     | YES  |                   |                |            |
+| [macChecksumSeed][10] | VARCHAR(40)  |            |     | YES  |                   |                |            |
+
+[1]: #build
+[2]: #majorVersion
+[3]: #minorVersion
+[4]: #bugfixVersion
+[5]: #hotfixVersion
+[6]: #winAuthSeed
+[7]: #win64AuthSeed
+[8]: #mac64AuthSeed
+[9]: #winChecksumSeed
+[10]: #macChecksumSeed
```

Comment:
```suggestion
[2]: #majorversion
[3]: #minorversion
[4]: #bugfixversion
[5]: #hotfixversion
[6]: #winauthseed
[7]: #win64authseed
[8]: #mac64authseed
[9]: #winchecksumseed
[10]: #macchecksumseed
```

Reply:
Only this need to be lowercase for it to work

Reply:
Okay, big thanks